README.md

```diff
@@ -180,6 +207,80 @@ ci16_sigmffile = collection.get_SigMFFile(stream_name='example_ci16')
 cf32_sigmffile = collection.get_SigMFFile(stream_name='example_cf32')
 ```
 
+### Create a SigMF Archive
+The `SigMFArchive` class, the `SigMFFile.archive()` method, and the
+`SigMFFile.tofile()` method can all be used to create an archive. 
+
+```python
+import numpy as np
+
+from sigmf.sigmffile import (SigMFFile,
+                             SigMFArchive)
+
+
+# create data file
+random_data = np.random.rand(128)
+data_path = "recording.sigmf-data"
+random_data.tofile(data_path)
+
+# create metadata
+sigmf_file = SigMFFile(name='recording')
+sigmf_file.set_global_field("core:datatype", "rf32_le")
+sigmf_file.add_annotation(start_index=0, length=len(random_data))
+sigmf_file.add_capture(start_index=0)
+sigmf_file.set_data_file(data_path)
+
+# create archive using SigMFArchive
+archive = SigMFArchive(sigmffiles=sigmf_file,
+                        path="single_recording_archive1.sigmf")
+
+# create archive using SigMFFile archive()
+archive_path = sigmf_file.archive(file_path="single_recording_archive2.sigmf")
+
+# create archive using tofile
+sigmf_file.tofile(file_path="single_recording_archive3.sigmf",
+                    toarchive=True)
+```
+
+### Create SigMF Archives with Multiple Recordings
+Archives with multiple recordings can be created using `SigMFArchive` class.
```

Comment:
Minor typo, missing the word "the" before "`SigMFArchive` class"

Reply:
Fixed in 1223ead